src/controllers/update/overseas.entity.review.controller.ts

```diff
@@ -31,7 +31,8 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
 export const post = (req: Request, res: Response, next: NextFunction) => {
   try {
     logger.debugRequest(req, `POST ${config.OVERSEAS_ENTITY_REVIEW_PAGE}`);
-    return res.redirect(config.UPDATE_CHECK_YOUR_ANSWERS_PAGE);
+    // return res.redirect(config.UPDATE_BENEFICIAL_OWNER_TYPE_PAGE);
```

Comment:
Currently directs to MO type page, will change in future when design has been agreed upon

Reply:
Would it be better to add this explanation as a comment above line 35 rather than just leaving code comments there?

When the design has been agreed upon, the person implementing it is more likely to look at the code than this pull request.

Also having sections of code commented out is usually seen as a code smell and I think will be picked up by SonarQube as that.

Reply:
Good idea, I'll add a TO-DO comment instead

Reply:
So TODO's will still be flagged as a code smell by SonarQube but as "Info" not "Major".

That's not as bad though I'm not sure what the threshold to get it to pass is. 

It might be bet just to put something like "If x is implemented then this will need to be changed to y" especially if we're not 100% sure it will be changed.

If not someone may attempt to do the TODO when they're not meant to

Reply:
Fixed

Reply:
Sorry I can't see the fix

Reply:
Ah sorry I think I made the comment before a successful push. Should be up now

Reply:
Cool